package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@lumigo/node-core": "1.16.0",
     "agentkeepalive": "^4.1.4",
-    "axios": "1.6.5",
+    "axios": "1.7.7",
```

Comment:
You didn't update the lock file, make sure you update it using node 14 (If you use a newer version of node you will create a lock file that works for a newer version, we want to make it match the lowest version we support)